src/components/DemoCarousel.tsx

```
@@ -148,27 +155,27 @@ const DemoCarousel: React.FC = () => {
               aria-label='Previous slide'
             >
-              <ChevronLeft className='w-5 h-5 text-gray-700' />
+              <ChevronLeft className='w-4 h-4 md:w-5 md:h-5 text-gray-700' />
             </button>
 
             <button
               onClick={nextSlide}
-              className='absolute right-4 top-1/2 -translate-y-1/2 p-2 bg-white/90 hover:bg-white rounded-full shadow-lg transition-all duration-200 hover:scale-110'
+              className='absolute right-2 md:right-4 top-1/2 -translate-y-1/2 p-2 md:p-3 bg-white/90 hover:bg-white rounded-full shadow-lg transition-all duration-200 hover:scale-110'
               aria-label='Next slide'
             >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)